R/import.R

```diff
@@ -107,8 +107,17 @@ import_objs <- function(
     }
   }
 
-  dep_table <- find_internal_deps(env)
-  all_dep_names <- c(names, unlist(dep_table[names], recursive = FALSE, use.names = FALSE))
+  if ("*" %in% names) {
+    explicit_names <- setdiff(names, "*")
+    all_names <- names(env)
```

Comment:
For future reference, this should be:

```suggestion
    all_names <- ls(env, all.names = TRUE)
```

I'll just merge and make the changes afterward.

Reply:
Thank you!